cilium-dbg/cmd/bpf_multicast_subscribers.go

```diff
@@ -0,0 +1,294 @@
+				return
+			}
+			Fatalf("Failed to open multicast bpf maps: %s", err)
+		}
+
+		var groups []netip.Addr
+		if all {
+			groups, err = groupV4Map.List()
+			if err != nil {
+				Fatalf("Failed to list multicast groups: %s", err)
+			}
+		} else {
+			groups = append(groups, groupAddr)
+		}
+
+		err = outputGroups(groupV4Map, groups)
+		if err != nil {
+			Fatalf("Failed to list multicast subscribers: %s", err)
+		}
+	},
+}
+
+var MulticastGroupSubscriberAddCmd = &cobra.Command{
+	Use:   "add <group> <subscriber-address>",
+	Short: "Add a remote subscriber to the given multicast group.",
+	Long: `Add a remote subscriber to the given multicast group.
+Only remote subscribers are added via command line. Local subscribers will be automatically populated in the map based on IGMP messages.
+Remote subscriber is typically other Cilium nodes. To add remote subscriber, following information is required:
+- group: multicast group address to which subscriber is added.
+- subscriber-address: cilium internal IP of remote node.
+
+For Example: 
+To add a remote node 10.100.0.1 to multicast group 229.0.0.1, use the following command:
+	cilium-dbg bpf multicast subscriber add 229.0.0.1 10.100.0.1
+`,
+	Run: func(cmd *cobra.Command, args []string) {
+		common.RequireRootPrivilege("cilium bpf multicast subscriber add")
+
+		link, err := netlink.LinkByName(defaults.VxlanDevice)
+		if err != nil {
+			Fatalf("Failed to get cilium_vxlan device %q: %s", defaults.VxlanDevice, err)
+		}
+
+		groupAddr, subIP, err := parseMulticastGroupSubscriberArgs(args)
+		if err != nil {
+			Fatalf("invalid argument: %s\n", err)
+		}
+
+		groupV4Map, err := maps_multicast.OpenGroupV4OuterMap(maps_multicast.GroupOuter4MapName)
+		if err != nil {
+			if errors.Is(err, fs.ErrNotExist) {
+				fmt.Fprintln(os.Stderr, "Cannot find multicast bpf maps")
+				return
+			}
+			Fatalf("Failed to open multicast bpf maps: %s", err)
+		}
+
+		subscriberMap, err := groupV4Map.Lookup(groupAddr)
+		if err != nil {
+			Fatalf("Failed to lookup multicast group %s: %s", groupAddr, err)
+		}
+
+		subscriber := &maps_multicast.SubscriberV4{
+			SAddr:    subIP,
+			Ifindex:  uint32(link.Attrs().Index),
+			IsRemote: true,
+		}
+
+		if err := subscriberMap.Insert(subscriber); err != nil {
+			Fatalf("Failed to add subscriber %s to multicast group %s: %s", subIP, groupAddr, err)
+		}
+	},
+}
+
+var MulticastGroupSubscriberDelCmd = &cobra.Command{
+	Use:     "delete <group> <subscriber-address>",
+	Aliases: []string{"del"},
+	Short:   "Delete a subscriber from the given multicast group.",
+	Long: `Delete a subscriber from the given multicast group.
+To delete remote subscriber, following information is required:
+- group: multicast group address from which subscriber is deleted.
+- subscriber-addr: subscriber IP address
+For Example: 
+To delete a remote node 10.100.0.1 from multicast group 229.0.0.1, use the following command:
+	cilium-dbg bpf multicast subscriber delete 229.0.0.1 10.100.0.1
```

Comment:
```suggestion
Example: `To delete a remote node 10.100.0.1 from multicast group 229.0.0.1, use the following command:

	cilium-dbg bpf multicast subscriber delete 229.0.0.1 10.100.0.1`
```
Same as for the add command.